codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSHttpBindingProtocolGenerator.kt

```diff
@@ -39,6 +39,10 @@ abstract class AWSHttpBindingProtocolGenerator : HttpBindingProtocolGenerator()
     override var serviceErrorProtocolSymbol: Symbol = AWSClientRuntimeTypes.Core.AWSServiceError
 
     override val unknownServiceErrorSymbol: Symbol = AWSClientRuntimeTypes.Core.UnknownAWSHTTPServiceError
+
+    override val retryErrorInfoProviderSymbol: Symbol
+        get() = AWSClientRuntimeTypes.Core.AWSRetryErrorInfoProvider
```

Comment:
When building for AWS, override the default error type that is code-generated into clients.